scss/icons/coin.scss

```
@@ -0,0 +1,20 @@
+$icon-coin-colors: (#060606, #ffffff, #ffc107);
```

stylelint will fix this code as follows.
```suggestion
$icon-coin-colors: (#060606, #fff, #ffc107);
```